Disable codesign for macosx cross-compile toolchain. Codesign only works on OSX.

Index: cmake/Modules/AddCompilerRT.cmake
===================================================================
--- a/compiler-rt/cmake/Modules/AddCompilerRT.cmake	(revision 342374)
+++ b/compiler-rt/cmake/Modules/AddCompilerRT.cmake	(working copy)
@@ -290,14 +290,6 @@
         set_target_properties(${libname} PROPERTIES IMPORT_PREFIX "")
         set_target_properties(${libname} PROPERTIES IMPORT_SUFFIX ".lib")
       endif()
-      if(APPLE)
-        # Ad-hoc sign the dylibs
-        add_custom_command(TARGET ${libname}
-          POST_BUILD  
-          COMMAND codesign --sign - $<TARGET_FILE:${libname}>
-          WORKING_DIRECTORY ${COMPILER_RT_LIBRARY_OUTPUT_DIR}
-        )
-      endif()
     endif()
     install(TARGETS ${libname}
       ARCHIVE DESTINATION ${install_dir_${libname}}
